databases/schemas/friends.sql

```diff
@@ -0,0 +1,14 @@
+CREATE TABLE IF NOT EXISTS friends (
```

Comment:
No need for `IF NOT EXISTS`. Don't run this code every connection. Migrations will require more than this.